ui/app/mainui/Popups.qml

```diff
@@ -241,6 +242,18 @@ QtObject {
                   })
     }
 
+    function openCommunityShareAddressesPopup(communityId, name, imageSrc) {
+        openPopup(communityIntroDialogPopup,
+                  {communityId: communityId,
+                   stackTitle: qsTr("Share addresses to %1").arg(name),
```

Comment:
```suggestion
                   stackTitle: qsTr("Share addresses with %1").arg(name),
```

Reply:
I think the name here is the community name. Maybe somthing like this works better
```suggestion
                   stackTitle: qsTr("Share addresses with %1's owner").arg(name),
```